Gemfile.lock

```diff
@@ -313,6 +313,9 @@ GEM
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2021.0225)
+    mimemagic (0.3.10)
```

Comment:
why did you add mimemagic as a dependency?

Reply:
from glancing at `Gemfile`, my take is that this came along for the ride with the other updates?

how would you feel about approving/merging this to keep things moving, and ticketing mimemagic cleanup for later?  or, are you worried this'll cause trouble when deployed?  i have no guess on that last point either way.

Reply:
@jmartin-sul it shouldn't take too long or be too hairy to pluck this out and try again, so nbd

Reply:
ah ok, cool

Reply:
@jmartin-sul @jcoyne done.

Reply:
approved again 🙂

Reply:
@jcoyne another look?